build.rs

```diff
@@ -7,78 +7,66 @@ use std::{
     str,
 };
 
-#[cfg(not(target_os = "macos"))]
-const LINUX_SDK_URL: &str  = "https://github.com/jabber-tools/cognitive-services-speech-sdk-rs-files/blob/main/SpeechSDK/1.22.0/linux/SpeechSDK-Linux-1.22.0.tar.gz?raw=true";
+const SPEECH_SDK_VERSION: &str = "1.36.0";
 
-#[cfg(not(target_os = "macos"))]
 fn download_file(url: &str, dst: &str) {
     Command::new("curl")
-        .args(&["-SL", url, "-o", dst])
+        .args(["-SL", url, "-o", dst])
         .status()
         .expect("failed to download Speech SDK!");
 }
 
-#[cfg(not(target_os = "macos"))]
+#[cfg(target_os = "linux")]
 fn main() {
-    let out_path = PathBuf::from(env::var("OUT_DIR").unwrap());
-    // copying SpeechSDK from local folder just worked fine but crates.io allows
-    // only 10 MB to be uploaded per crate. Because of that SpeechSDK shared libraries
-    // were moved to separate repository (https://github.com/jabber-tools/cognitive-services-speech-sdk-rs-files)
-    // and must be downloaded during build from there.
-    /*
-    fs::copy(
-        "./SpeechSDK/lib/x64/libMicrosoft.CognitiveServices.Speech.core.so",
-        Path::new(&out_path).join("libMicrosoft.CognitiveServices.Speech.core.so"),
-    )
-    .unwrap();
-    fs::copy(
-        "./SpeechSDK/lib/x64/libMicrosoft.CognitiveServices.Speech.extension.kws.so",
-        Path::new(&out_path).join("libMicrosoft.CognitiveServices.Speech.extension.kws.so"),
-    )
-    .unwrap();
-    fs::copy(
-        "./SpeechSDK/lib/x64/libMicrosoft.CognitiveServices.Speech.extension.codec.so",
-        Path::new(&out_path).join("libMicrosoft.CognitiveServices.Speech.extension.codec.so"),
-    )
-    .unwrap();
-
-    println!("cargo:rustc-link-search=native={}", out_path.display());
-    println!("cargo:rustc-link-lib=dylib=Microsoft.CognitiveServices.Speech.core");
-    */
+    let linux_sdk_url = format!(
+        "https://csspeechstorage.blob.core.windows.net/drop/{SPEECH_SDK_VERSION}/SpeechSDK-Linux-{SPEECH_SDK_VERSION}.tar.gz");
```

Comment:
I would like to understand what kind of service is this. Is this some public storage service or something you own?

I was able to download macos file manually but no linux one.

Originally linux files were  in my repo (https://github.com/jabber-tools/cognitive-services-speech-sdk-rs-files) but I was unable to push there mac os files since they were too big. I was thinking about some other option and ended up with dropbox. 

I am willing to go to something better but as library owner I need to have control over binary storage

Also where speech sdk was downloaded from? I remember Microsoft provides only latest version which is awkward, ideally i would like to store all the versions somewhere as this library will be gradually upgraded to use newer vesions of speech sdk

Reply:
btw. I don't see windows link, is windows supported? right now i don't even remember if I was able to build this on windows.

Reply:
> I would like to understand what kind of service is this. Is this some public storage service or something you own?

No, it's not my own, this is on the Microsoft server, this is what the SDK download links in the [official docs](https://learn.microsoft.com/en-us/azure/ai-services/speech-service/quickstarts/setup-platform?tabs=macos%2Cubuntu%2Cdotnetcli%2Cdotnet%2Cjre%2Cmaven%2Cnodejs%2Cmac%2Cpypi&pivots=programming-language-cpp) actually pointed to.

For Linux:
https://aka.ms/csspeech/linuxbinary  got resolved into
https://csspeechstorage.blob.core.windows.net/drop/1.36.0/SpeechSDK-Linux-1.36.0.tar.gz

The macOS speech SDK link is the same: 
https://aka.ms/csspeech/macosbinary eventually got resolved into this link:
https://csspeechstorage.blob.core.windows.net/drop/1.36.0/MicrosoftCognitiveServicesSpeech-MacOSXCFramework-1.36.0.zip

Reply:
>  I don't see windows link, is windows supported?

Not in this PR, I'm planning to add Windows support in another PR.